spec/models/email_link_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'rails_helper'
+
+RSpec.describe EmailLink, type: :model do
+    it "default factory is valid" do
+       expect(build(:email_link).valid?).to be_truthy
+
+
+    end
+
+    it "fails validations with invalid attributes" do
+        expect(build(:email_link, url:"").valid?).to be_falsy
+    end
+
+    it "returns undecided emails" do
+
+        email1 = build(:email_link)
+        email2 = build(:email_link, accept_or_rejected_at: DateTime.now, accepted: true)
```

Comment:
On line 11 we tried to use a trait that built an invalid email, the factory girl linter threw an error. It seems that factory girl tried to save factories to the database. Does that mean we cannot use traits to build invalid models for testing?